siliconcompiler/libs/asap7sc7p5t.py

```diff
@@ -0,0 +1,91 @@
+import os
+import sys
+import re
+import siliconcompiler
+
+def make_docs():
+    '''
+    Library description
+
+    Documentation: https://
+    Sources: https://
+    Installation: https://
+    '''
+
+    chip = siliconcompiler.Chip()
+    setup(chip)
+    return chip
+
+def setup(chip):
+
+    foundry = 'virtual'
+    process = 'asap7'
+    libtype = '7p5t'
+    rev = 'r1p7'
+    corner = 'typical'
+    objectives = ['setup']
+
+    all_libs = {
+        'asap7sc7p5t_rvt' : 'R',
+        'asap7sc7p5t_lvt' : 'L',
+        'asap7sc7p5t_slvt' : 'SL'
+    }
+
+    for libname in all_libs.keys():
+        libdir = os.path.join('..', 'third_party', 'pdks', foundry, process, 'libs', libname, rev)
+        suffix = all_libs[libname]
+
+        # rev
+        chip.set('library',libname, 'package', 'version',rev)
+
+        # timing
+        chip.add('library', libname, 'nldm', corner, 'lib',
```

Comment:
Should we be using `.add` here, or `.set`? Same for other calls later in this file.